docs/finality-provider-operation.md

```diff
@@ -906,18 +906,19 @@ Parameters:
 
 * `<address>`: The Babylon address of the stakeholder in bech32 string.
 
-#### 6.8 Recover local status of a finality provider
+#### 6.7.2 Set Withdraw Address
```

Comment:
I was thinking maybe we should put `Withdraw Rewards` ahead of `Set Withdraw Address` as the latter is optional.